### ISSUE TITLE
[a11y] context menu is missing semantics role

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#161561]
When using this widget https://api.flutter.dev/flutter/material/TextSelectionToolbar-class.html and https://api.flutter.dev/flutter/widgets/ContextMenuController-class.html, the widget is missing following role

| OS | role |
|--------|--------|
| web | toolbar |
| ios | - |
| macos | NSAccessibilityToolbarRole |
| windows | ROLE_SYSTEM_TOOLBAR |
| android | ContextMenu | 

The item in the context menu 

| OS | role |
|--------|--------|
| web | button |
| ios | - |
| macos | NSAccessibilityButtonRole, NSAccessibilityToolbarButtonSubrole |
| windows | ROLE_SYSTEM_PUSHBUTTON |
| android | - |
